### DISCUSSION TITLE
E1 Outdoor - problem after upgrade camera to firmware v3.1.0.804_22011506

### DISCUSSION THREAD

**Discussion by ArcziHA** [fwestenberg/reolink_dev#477] [Q&A]
Does anyone have a problem with the camera image after the required update? The current version is v. 3.1.0.804_22011506. Tried adding the integration again, it works, but still no picture.

**Comment by Swiftrail84** [comment]
same here with RLC-510WA.

https://github.com/fwestenberg/reolink_dev/issues/486
